config/mtl.toml

```diff
@@ -240,3 +240,43 @@ count = 8
 				13, 0, 0, 0, 20480, 36140000, 16, 96, 0, 0, 0,
 				14, 0, 0, 0, 20480, 38003000, 16, 68, 0, 0, 0,
 				15, 0, 0, 0, 20480, 39787000, 45, 51, 0, 0, 0] 
+
+	[[module.entry]]
+	name = "SRC"
+	uuid = "E61BB28D-149A-4C1F-B709-46823EF5F5AE"
+	affinity_mask = "0x1"
+	#instance_count = "10"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "7"
+	auto_start = "0"
+	sched_caps = [1, 0x00008000]
+
+	# pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xffff, 0xc, 0x8, 0x05ff,
+			1, 0, 0xf6c9, 0xc, 0x8, 0x05ff]
+
+	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
+	mod_cfg = [0, 0, 0, 0, 12832, 1365500, 0, 0, 0, 1365, 0,
+				1, 0, 0, 0, 12832, 2302300, 0, 0, 0, 2302, 0,
+				2, 0, 0, 0, 12832, 3218200, 0, 0, 0, 3218, 0,
```

Comment:
I'm not really sure, those values were taken from closed source

Reply:
@singalsu how can we generate this data ?
@kfrydryx is this needed for MTL or can it merge later with the tooling ?

Reply:
@lgirdwood That's needed to enable SRC module on mtl. With this config it is (or at least was) running.

Reply:
@kfrydryx ok, but we need to understand how this is generated so that it can be updated if needed. e.g. do we have a Python or Matlab script that does this ? Btw, do we have a description of the fields anywhere ?

Reply:
@lgirdwood I have no idea what this data is. @kfrydryx Is this for open-source SRC in SOF or for something else?

Reply:
@lgirdwood @singalsu Checked on with the team, got a bit of clarification, but not the full answer. Those values are supposed to be used by the driver to build module configuration data. For the most part it should be desired to keep them the same as in ace, though minor tweaks might be necessary if module implementation differs from ace one. 
For the current state of things I believe they are not used in any functional way. I suppose we could treat them as initial/reference values
@marcinszkudlinski We might have to verify all of the mod_cfg tables. I suspect at least some of them were also copy-pasted from ace without further consideration.

- PAR_0-3 - configuration lookup parameters used by the driver
- IS_BYTES - actual size (in bytes) of instance .bss if configured by this entry
- CPS - number of DSP cycles required by the module instance to process one second of audio
- IBS - input buffer size (might be obsolete?)
- OBS - output buffer size (might be obsolete?)
- MOD_FLAGS - currently unused
- CPC - number of DSP cycles to process one frame of data
- OBLS - output block size, currently not used

Reply:
@kfrydryx thanks for the info, looks like we will need to document this i.e. what is needed (for FW and SW), how to calc etc.
@singalsu and I have had a chat and we think we can add a UT Kconfig option that would allow FW to enter a test mode and dump these stats for each module/config.

Reply:
@lgirdwood @singalsu Will you do this, or should we try? Also, can this be merged as an initial configuration to be tweaked later? Without this SRC is not run under our CI.

Reply:
@kfrydryx We don't have yet the system to measure these parameters so I feel it's better to proceed with this PR since it fixes an issue for you and we can add a measured version later.

Reply:
ok, lets merge this to unblock CI and figure out how we do the tooling around the module data (and what data is needed etc)..